node/lib/package/LocalPackageSource.ts

```diff
@@ -319,14 +320,112 @@ export class LocalPackageSource extends PackageSource {
                 onboarding: onboardingModulePath,
                 onboardingProperties: onboardingProperties,
             }],
+            onboardingInfo: [],
             version: packageJson.version,
         };
     }
 
     private async loadOnboardingPackageInfoAsync(
-            onboardingName: string): Promise<PackageInfo | null> {
-        // TODO: Load onboarding package info from XML + package.json
-        return Promise.resolve(null);
+        onboardingName: string): Promise<PackageInfo | null> {
+
+        // By convention there are manifest.xml and package.json files under 'node'' directory. 
+        // under a directory with the translator name, under the schema directory.
```

Comment:
sure. missed them.

---
In reply to: [75263892](https://github.com/openT2T/opent2t/pull/22#discussion_r75263892) [](ancestors = 75263892)